Cargo.toml

```diff
@@ -26,8 +22,15 @@ tide-disco = "0.9"
 toml = "0.8"
 tracing = "0.1"
 vbs = "0.1.4"
+tokio = { version = "1", default-features = false, features = [
```

Comment:
also what is the reason to just use tokio now and remove async-std?

Reply:
- We only have to support a single executor in HotShot
- It lets us use async primitives unsupported by the compatibility layer
- Development would no longer involve manually setting RUSTFLAGS everywhere
- It lets us migrate off of tide to a more modern Tokio-based framework (we can technically do this now if we rock `tokio1`, but I think it's best to have everything on a single executor)

In addition to the above, Tokio is better maintained